src/components/discovery/LocusSearch.js

```diff
@@ -25,13 +27,16 @@ const LocusSearch = ({assemblyId, addVariantSearchValues}) => {
         const result = parse.exec(value);
 
         if (!result) {
+            setCurrentLocus({chrom: null, start: null, end: null});
+            setLocusValidity(false);
             return;
         }
 
         const chrom = result[1].toUpperCase(); //for eg 'x', has no effect on numbers
         const start = Number(result[2]);
         const end = Number(result[3]);
 
+        setCurrentLocus({chrom: chrom, start: start, end: end});
```

Comment:
this can be shortened to `{chrom, start, end}` - and the same object is used for `addVariantSearchValues` as well

Reply:
honestly I always found this style confusing